examples/python/evm/scoped_account.send_transaction_custom.py

```diff
@@ -0,0 +1,39 @@
+# Usage: uv run python evm/scoped_account.send_transaction_custom.py
```

Comment:
This example should be moved to fit into the new example structure – under `python/managed-mode`